cfpb/cfgov-refresh
# Leadership Calendar: Back to Django

This brings the leadership calendar back into plain-old-Django, and removes the Sheer processor. This will make lives easier in a number of ways: content managers won't have to worry about the "sheer index", and dev team should see less unexplainable test failures in Travis and elsewhere (because the processor was so flaky).

There's also been some simplification around PDF generation:
- We're passing rendered HTML to PDFReactor, instead of A URL (that it would need to turn around and fetch). This should reduce the amount of different ways PDF generation can fail. In other words, the current path is: Django  ->  PDFReactor -> Django, with this PR it's just Django-> PDFReactor.
- The download form now uses GET instead of POST, meaning the generated PDF's should be cachable in Akamai.

Also, somewhere along the way we broke [this page](http://www.consumerfinance.gov/credit-cards/knowbeforeyouowe/) and nobody noticed. I've re-added that URL in this PR. (cc @schaferjh )

## Testing

- `pip uninstall leadership-calendar`
- Load this SQL into your DB: [calendar.sql.zip](https://github.com/cfpb/cfgov-refresh/files/321190/calendar.sql.zip)
- Check out http://localhost:8000/about-us/the-bureau/leadership-calendar/ and try filtering by person and date.
- Try the download form too-- if you are on a machine without PDFReactor (or PDFReactor is failing somehow), it should fail over to showing you the HTML representation (inspired by @kave's #2193)

## Review

- @kurtw @richaagarwal 


## Todos

- see how this interacts with the work in #2188

## Checklist
